GliaWidgets/Sources/View/Chat/SecureMessagingBottomBannerViewStyle.RemoteConfig.swift

```diff
@@ -1,3 +1,43 @@
-// TODO: Unified customization will be implemented with MOB-3717
+import UIKit
 
-import Foundation
+extension SecureMessagingBottomBannerViewStyle {
+    mutating func apply(
+        configuration: RemoteConfiguration.SecureConversations?,
+        assetBuilder: RemoteConfiguration.AssetsBuilder
+    ) {
+        configuration?.bottomBannerBackground?.color.unwrap {
+            switch $0.type {
+            case .fill:
+                $0.value
+                    .map { UIColor(hex: $0) }
+                    .first
+                    .unwrap { backgroundColor = .fill(color: $0) }
+            case .gradient:
+                let colors = $0.value.convertToCgColors()
+                backgroundColor = .gradient(colors: colors)
+            }
+        }
+
+        configuration?.topBannerDividerColor?.value
+            .map { UIColor.init(hex: $0) }
+            .first
+            .unwrap {
+                self.dividerColor = $0
```

Comment:
I would keep `.unwrap { self. dividerColor = $0 }` in one line so it would be the same as above and it looks much cleaner. Just a matter of code style.

Reply:
Fixed within 4fe3105570659aa9e020d60ee26e894a383507cc.